Application/Features/MapFeatures/Queries/GetAllMapQuery.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using Application.Interfaces;
+using AutoMapper;
+using AutoMapper.QueryableExtensions;
+using MediatR;
+
+namespace Application.Features.MapFeatures.Queries
+{
+	public class GetAllMapQuery:IRequest<Responce>
+	{
+
+
+	}
+
+    public class GetAllMapQueryHandler : IRequestHandler<GetAllMapQuery, Responce>
+    {
+        private readonly IApplicationDbContext _context;
+        private readonly IMapper _mapper;
+
+        public GetAllMapQueryHandler(IApplicationDbContext context, IMapper mapper)
+        {
+            _context = context;
+            _mapper = mapper;
+        }
+
+        public async Task<Responce> Handle(GetAllMapQuery request, CancellationToken cancellationToken)
+        {
+            var responce =  _context.Maps.ProjectTo<MapDto>(_mapper.ConfigurationProvider).ToList();
```

Comment:
it's better to add await and at the end call method ToListAsync(cancellationToken);